src/sentry/discover/arithmetic.py

```diff
@@ -53,6 +57,15 @@ def validate(self) -> None:
         if self.operator == "divide" and self.rhs == 0:
             raise ArithmeticValidationError("division by 0 is not allowed")
 
+    def to_snuba_json(self, alias: Optional[str] = None) -> JsonQueryType:
+        """ Convert this tree of Operations to the equivalent snuba json """
+        lhs = self.lhs.to_snuba_json() if isinstance(self.lhs, Operation) else self.lhs
+        rhs = self.rhs.to_snuba_json() if isinstance(self.rhs, Operation) else self.rhs
+        result = [self.operator, [lhs, rhs]]
```

Comment:
Since it's only allowing for 1 operation right now this should be okay but something to be wary of when allowing for more deeply nested expressions is the following

```
["plus", ["transaction.duration", ["plus", ["spans.http", "measurements.lcp"]]]]
```

Because snuba _may_ interpret `transaction.duration` as a function call.

Reply:
:+1: In testing I think this worked with fields but good reminder if I see this fail 🙃